src/main/java/liqp/Template.java

```diff
@@ -569,7 +576,7 @@ public String renderUnguarded(Map<String, Object> variables, TemplateContext par
                 contextHolder.setContext(templateContext);
             }
             Object rendered = node.render(this.templateContext);
-            return rendered == null ? "" : String.valueOf(rendered);
+            return (rendered == null || rendered == "null") ? "null" : String.valueOf(rendered);
```

Comment:
This doesn't look clean , we are going to return always "null" ? 
or rendered string

why null needs to be returned as "null" ?

Reply:
correct I think we can make it
return rendered == null ? "null" : String.valueOf(rendered);  or if we want empty, we can make it as
return rendered == null ? "" : String.valueOf(rendered);

and remove this: || rendered == "null"

we are anyways catching it in "render()":
catch(Exception e) {
            if (context.renderSettings.showExceptionsFromInclude) {
                throw new RuntimeException("problem with evaluating include", e);
            } else {
                return "";
            }
        }